core/arch/mangle_shared.c

```diff
@@ -1824,7 +1824,7 @@ find_syscall_num(dcontext_t *dcontext, instrlist_t *ilist, instr_t *instr)
 
```

Comment:
Please add a regression test: any sanity test of -inject_x64 will do.

Reply:
I will do.